dist/index.html

```diff
@@ -22,8 +22,8 @@
         <!-- Add your site or application content here -->
         <p>Hello world! This is HTML5 Boilerplate.</p>
 
-        <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>
-        <script>window.jQuery || document.write('<script src="js/vendor/jquery-1.11.2.min.js"><\/script>')</script>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
```

Comment:
I was holding off on creating a PR myself as this is still 404'ing. Given that quite many people use master, that may cause confusion. The team sent mails to the google CDN guys so generally this will be fixed quick.